### PR TITLE
Fix render error

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

File: console/app/controllers/application_types_controller.rb [hunk 1]
```diff
@@ -138,7 +138,8 @@ def estimate
 
     render :inline => gear_increase_indicator(cartridges, scales, application.gear_profile, false, capabilities, usage_rates, owner)
```
Comment (joelsmith) [inline console/app/controllers/application_types_controller.rb]: I'd like to see the :inline changed to :text here too (and in the other file too) unless we expect that the output of gear_increase_indicator() would ever be ERB that we want to evaluate. From my reading of it, it shouldn't ever be.

Comment (liggitt) [inline console/app/controllers/application_types_controller.rb]: yup, updated and made sure the indicator still appeared correctly

**Comment by Miciah** [comment]
Looks good.  [merge].

**Comment by Miciah** [comment]
Failure looks spurious (errors in PHP cartridge's manifest).  [merge] again.

**Pull request merged**